docs/Design/Lists/Native List Commands.md

```diff
@@ -0,0 +1,103 @@
+# Native List Commands
+
+## Feature Overview
+
+All supported languages provide some amount of built-in list support to interact with the contents of a list.

```

Comment:
> built-in list support to interact with the contents of a list
> built-in list support

You can probably remove the first "list" in there, for just "built-in support". Seems redundant.